docker-compose/readme.md

```diff
@@ -462,37 +462,20 @@ You can run selenium tests against a local bb2 server by following below steps:
    ./docker-compose/run_selenium_tests_local.sh account
    ```
 
-3. To debug tests (visualize browser interaction): point VNC client to localhost:5900
-
-   1. requires installation of vnc viewer, password (secret)
-   2. Start tests using -d option as shown below:
-
-   use MSLSX (default)
-
-   ```
-   ./docker-compose/run_selenium_tests_local.sh -d
-   ```
-
-   ```
-   ./docker-compose/run_selenium_tests_local.sh -d mslsx
-   ```
-
-   use SLSX
-
-   ```
-   ./docker-compose/run_selenium_tests_local.sh -d slsx
-   ```
-
 ## Running Selenium tests on remote BB2 server (SBX, TEST, PROD)
 
 You can run selenium tests against a remote bb2 server by following below steps:
 
 From the base directory of the local repo run:
 ```
-./docker-compose/run_selenium_tests_remote.sh -d SBX
+./docker-compose/run_selenium_tests_remote.sh -p SBX/TEST
```

Comment:
Not huge, but we could reduce these options to one line like `[SBX | TEST | PROD]` just to avoid any confusion or someone entering SBX/TEST literally. I could imagine that given the way it contrasts to the PROD command below it.

Reply:
The `[SBX | TEST]` format is definitely better, but this is intentionally split out between SBX/TEST and PROD because the `-p` option is needed for SBX/TEST and will break things in PROD. We should probably edit this so the script figures that out based on environment. Maybe that can be a small tech debt type ticket we can bring in if we need things to fill out the sprint.